src/components/PeriodPopover/styles.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import Calendar from '../Calendar/Calendar';
```

Comment:
You don't need to write `Calendar/Calendar` you have index file inside Calendar filder. So, just `import Calendar from '../Calendar';` will be enough.